lib/Serverless.js

```diff
@@ -68,13 +68,13 @@ class Serverless {
   run() {
     if (!this.cli.displayHelp(this.processedInput) && this.processedInput.commands.length) {
       // trigger the plugin lifecycle when there's something which should be processed
-      return this.pluginManager.run(this.processedInput.commands);
+      this.pluginManager.run(this.processedInput.commands);
```

Comment:
I'm a little nervous about this :/ ... I feel that it might bring back some Async issues we've had before. Are you sure this is super necessary?